test/test.mli

```diff
@@ -1,11 +1,13 @@
 (*Ideally we should have a unified interface of testing framework, I am still thinking about how to abstract away the difference between JS testing and OUnit.*)
```

Comment:
Is there a reason why you're not just using bs-jest? I've considered making bs-ava too, so if there's a good reason to use it here, maybe we should create that?